universal_transfer_operator/docs/universal_transfer_operator/gcs.rst

```diff
@@ -0,0 +1,109 @@
+********************
+Google Cloud storage
+********************
+
+Transfer to Google Cloud storage as destination dataset
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+User can transfer data to Google cloud storage as destination as from following sources dataset:
+
+#. :ref:`table`
+    - Sqlite
+    - Snowflake
+    - Bigquery
+
+#. :ref:`file`
+    - AWS S3
+    - Google cloud storage
+    - Local
+    - SFTP
+
+Following transfer modes are supported:
+
+1. Transfer using non-native approach
+    Following is an example of non-native transfers between AWS S3 and Google cloud storage using non-native transfers:
+
+    .. literalinclude:: ../../example_dags/example_universal_transfer_operator.py
+       :language: python
+       :start-after: [START transfer_non_native_s3_to_gs]
+       :end-before: [END transfer_non_native_s3_to_gs]
+
+2. Transfer using native approach
```

Comment:
I think we should remove the transfers we are not yet supporting.

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1861/commits/e6359c247b1ac8547febf6a55ba1c5c15905a97e)